README.md

```diff
@@ -1,30 +1,10 @@
-# React + TypeScript + Vite
-
-This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
+# Virtual Field Environments
 
+A Virtual Field Environment is an interactive, browser based application that allows users to view a virtual environment through the use of photospheres.
 Currently, two official plugins are available:
 
-- [@vitejs/plugin-react](https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react/README.md) uses [Babel](https://babeljs.io/) for Fast Refresh
-- [@vitejs/plugin-react-swc](https://github.com/vitejs/vite-plugin-react-swc) uses [SWC](https://swc.rs/) for Fast Refresh
-
-## Expanding the ESLint configuration
-
-If you are developing a production application, we recommend updating the configuration to enable type aware lint rules:
-
-- Configure the top-level `parserOptions` property like this:
+To find documentation on this project please look at the projects [Wiki](https://github.com/kingsawpdx/virtualFieldEnvironments/wiki)
 
-```js
-export default {
-  // other rules...
-  parserOptions: {
-    ecmaVersion: "latest",
-    sourceType: "module",
-    project: ["./tsconfig.json", "./tsconfig.node.json"],
-    tsconfigRootDir: __dirname,
-  },
-};
-```
+For developer information: [click here](https://github.com/kingsawpdx/virtualFieldEnvironments/wiki/Developer-Guide)
 
-- Replace `plugin:@typescript-eslint/recommended` to `plugin:@typescript-eslint/recommended-type-checked` or `plugin:@typescript-eslint/strict-type-checked`
-- Optionally add `plugin:@typescript-eslint/stylistic-type-checked`
-- Install [eslint-plugin-react](https://github.com/jsx-eslint/eslint-plugin-react) and add `plugin:react/recommended` & `plugin:react/jsx-runtime` to the `extends` list
+![Screenshot of VFE](https://imgur.com/a/eRcKsQf)
```

Comment:
It's having a problem rendering the image, I think you have to [include the file extension in the link](https://stackoverflow.com/a/44989845)

Reply:
Alternatively, the image can be uploaded to the repo and referenced that way.